src/components/application_manager/include/application_manager/command_factory.h

```diff
@@ -54,6 +58,101 @@ class CommandFactory {
   virtual CommandSharedPtr CreateCommand(
       const commands::MessageSharedPtr& message,
       commands::Command::CommandSource source) = 0;
+  virtual bool IsAbleToProcess(
+      const int32_t,
+      const application_manager::commands::Command::CommandSource) const = 0;
+};
+
+class ICommandCreator {
+ public:
+  virtual ~ICommandCreator() {}
+  virtual bool isAble() const = 0;
```

Comment:
@BSolonenko please give more appropriate name (CanBeCreated for example) of this function and add doxygen comment

Reply:
@LuxoftAKutsan fix name and gave comment  in https://github.com/LuxoftAKutsan/sdl_core/pull/275/commits/75224d1cd9b7d4fc21af67229e902ff04fdaedb9